MAINTAINERS file cleanup

icecrime opened the pull request:
First phase in cleaning up the MAINTAINERS file to make it better reflect reality by:
- Removing "Operators" that have no practical role.
- Removing "Subsystems" as they are separate repositories with  their own MAINTAINERS files.

Overall, the goal of this first PR is to make the file more Engine specific. All the names are preserved until we can move the single source of truth to a different repository.

Contributes to #18319.

Signed-off-by: Arnaud Porterie arnaud.porterie@docker.com

Commit:
MAINTAINERS file cleanup

First phase in cleaning up the MAINTAINERS file to make it better
reflect reality by:

- Removing "Operators" that have no practical role.
- Removing "Subsystems" as they often are separate repositories with
  their own MAINTAINERS files.

Signed-off-by: Arnaud Porterie <arnaud.porterie@docker.com>

thaJeztah commented on MAINTAINERS:
```diff
@@ -271,127 +234,18 @@ made through a pull request.
+    # TODO Describe the docs maintainers role.
+
+        people = [
+            "james",
```
oh, while you're at it, can you remove "james", because it's a duplicate of "jamtur01" :smile:

icecrime replied:
Ha!

thaJeztah commented on MAINTAINERS:
```diff
@@ -271,127 +234,18 @@ made through a pull request.
-		[Org.Subsystem.builder]
-
-			people = [
-				"duglin",
```
Careful; doug was never added to Core Maintainers, so better add him to that list

thaJeztah replied:
oh, and I see Brian also never was added to one of the lists :smile:

jfrazelle replied:
+1

icecrime replied:
I noted that, but I think it's a different PR ;-)

thaJeztah replied:
Alright, sgtm!

duglin replied:
erikh too

thaJeztah replied:
https://github.com/docker/docker/pull/18321/files#diff-bb5907b965b501e1856ccd79d2c4f642R223

duglin replied:
ah cool

> On Nov 30, 2015, at 7:03 PM, Sebastiaan van Stijn <notifications@github.com> wrote:
> 
> In MAINTAINERS <https://github.com/docker/docker/pull/18321#discussion_r46223327>:
> 
> > -
> > -			people = [
> > -				"bfirsh",
> > -				"ehazlett"
> > -			]
> > -
> > -		[Org.Subsystem.compose]
> > -
> > -			people = [
> > -				"aanand"
> > -			]
> > -
> > -		[Org.Subsystem.builder]
> > -
> > -			people = [
> > -				"duglin",
> https://github.com/docker/docker/pull/18321/files#diff-bb5907b965b501e1856ccd79d2c4f642R223 <https://github.com/docker/docker/pull/18321/files#diff-bb5907b965b501e1856ccd79d2c4f642R223>
> —
> Reply to this email directly or view it on GitHub <https://github.com/docker/docker/pull/18321/files#r46223327>.
>

jfrazelle commented:
LGTM

tianon commented on MAINTAINERS:
```diff
@@ -271,127 +234,18 @@ made through a pull request.
-
-			people = [
-				"shykes",
-				"tianon"
```
Does this mean I get to be moved to the `core` list instead? :innocent:

jfrazelle replied:
tianon is not in core list either so add him there

jfrazelle replied:
jynx whoa

thaJeztah replied:
`[Org."<3"]`

tiborvass commented:
LGTM after fixing duplicate james

Commit:
Update core maintainers, remove duplicates

Move cpuguy83, duglin, tianon to core maintainers to reflect reality.

Remove james (duplicate for jamtur01) and add [people.jamtur01] section.

Signed-off-by: Arnaud Porterie <arnaud.porterie@docker.com>

icecrime commented:
Ok this made no sense in many ways :-) Added @cpuguy83 @duglin @tianon in core maintainers (because: come on), and fix the dupe.

thaJeztah commented:
LGTM, welcome @cpuguy83 @duglin @tianon ! :innocent:

jfrazelle commented:
LGTM

tiborvass commented:
I didn't even realize they weren't part of core maintainers

jfrazelle commented:
Same here, the were in our hearts

cpuguy83 commented:
<3 LGTM

tianon commented:
:metal: :heart: LGTM

estesp commented:
I don't **think** this file edit is going to break windows and that CI run looks stuck, so L G T M and I'll merge.. :)

thaJeztah commented:
tsk tsk

vdemeester commented:
:metal: yay :heart_eyes_cat:
